src/Components/CommentCard.jsx

```diff
@@ -0,0 +1,25 @@
+export default function CommentCard({comment, user}) {
+  return (
+    <>
+      <div className="flex flex-col">
+        <div className="border rounded-md p-3 ml-3 my-3 mx-3">
+          <div className="flex gap-3 items-center">
+            <img 
+            // src={user.avatar_url}
+                className="object-cover w-8 h-8 rounded-full 
+                border-2 border-indigo-400  shadow-indigo-400
+                "/>
+            <h3 className="font-bold">
+              {comment.author}
+              <br></br>
+              {comment.created_at}
```

Comment:
It would be better to have the timestamp formatted in a bit more of a human-readable way, than the default. Have a think about how you could manipulate the data yourself to do this or look into some packages that can do it for you 💡